assets/css/components/chirp-box.scss

```diff
@@ -0,0 +1,25 @@
+.chirp-box {
+  background: white;
```

Comment:
Use the variable `$white` for consistency.
Import mo muna si `common.scss`, refer the other stylesheets for reference.